source/operations/concepts/healing.rst

```diff
@@ -0,0 +1,110 @@
+.. _minio-concepts-healing:
+
+==============
+Object Healing
+==============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+What is healing?
+----------------
+
+Healing is MinIO's ability to restore an object that has been damaged, corrupted, or partially lost.
+The loss can come from random corruptions in the form of 
+- :ref:`bit rot <minio-ec-bitrot>`
+- the loss of one or more drives due to failure or outages
+- the corruption or loss of one or more shards of an object for other reasons
```

Comment:
I wonder if we should order these based on likelihood:

- Drive-level errors or failure
- OS/Filesystem errors or failure 
- :ref:`bit rot <minio-ec-bitrot>` of stored data

Reply:
and use the 'including but not limited to' preface.

Reply:
Yes, good ideas on both counts